### ISSUE TITLE
[CoE Starter Kit - Prod Environment in Spanish] 

### ISSUE THREAD

**Issue by lucasubieta** [microsoft/coe-starter-kit#8642]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Hi there!

We did an installation of the CoE, but we just notice that the prod environment where the CoE was deployed is in Spanish, not in English as the Microsoft documentation recommends.

Is there any known issue with this? Should we install the CoE again in a new English environment?
No errors so far but just to be sure..

Thanks a lot.

### What solution are you experiencing the issue with?

None

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by Jenefer-Monroe** [comment]
Yes the language of the CoE install environment needs to be English. I've not tested in other languages but do know others have posted bugs here that can only be addressed by having the language be English for the coe envt itself. 

I dont have a list of issues but its things like formula issues and power automate expressions that break.

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team
